### PR TITLE
Added Gradle TOML file with dependencies declarations

### PR DESCRIPTION
**Pull request by siarhei-luskanau**

### CODE REVIEW

**Comment by ArnyminerZ** [comment]
I don't know if we want to make this change, it would break how versions are declared for all the other bitfire libraries. What do you think @rfc2822 ?

**Comment by rfc2822** [comment]
What does this change achieve?

**Comment by siarhei-luskanau** [comment]
1. TOML catalog is supported by IntelliJ IDEA and Android studio. IntelliJ IDEA can navigate to dependency declaration and show if dependency has new version.
2. TOML catalog can be used in Gradle Groovy and Gradle Kotlin scripts.
3. TOML file can be shared between root project and buildSrc project. Also TOML file can be shared between several projects.

So TOML catalog is good tool to declare dependencies in single common place.

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[versions]
+
```
Comment (rfc2822) [inline gradle/libs.versions.toml:2]: Do we need these empty lines after the section? The examples on https://toml.io/en/ don't have them

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+commons-lang3 = "3.8.1" # // last version that doesn't require Java 8
```
Comment (rfc2822) [inline gradle/libs.versions.toml:3]: Please only use `#` as comment sign

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+commons-lang3 = { module = "org.apache.commons:commons-lang3", version.ref = "commons-lang3" }
+junit4 = { module = "junit:junit", version.ref = "junit4" }
+okhttp = { module = "com.squareup.okhttp3:okhttp", version.ref = "okhttpVersion" }
+okhttp-mockwebserver = { module = "com.squareup.okhttp3:mockwebserver", version.ref = "okhttpVersion" }
+xpp3 = { module = "org.ogce:xpp3", version.ref = "xpp3Version" }
+
+[plugins]
+
+kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
+dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
```
Comment (rfc2822) [inline gradle/libs.versions.toml:21]: Please use alphabetical order

**Review by rfc2822 — DISMISSED** [review]
Ah yes, thanks, makes sense! See also https://developer.android.com/build/migrate-to-catalogs

File: gradle/libs.versions.toml (hunk 1 above)
Comment (siarhei-luskanau) [reply gradle/libs.versions.toml]: Empty lines are removed

File: gradle/libs.versions.toml (hunk 2 above)
Comment (siarhei-luskanau) [reply gradle/libs.versions.toml]: Comment sign  is updated

File: gradle/libs.versions.toml (hunk 3 above)
Comment (siarhei-luskanau) [reply gradle/libs.versions.toml]: Sorted

**Pull request merged**